packages/dev-middleware/src/types/Experiments.js

```diff
@@ -23,6 +23,12 @@ export type Experiments = $ReadOnly<{
    * interface.
    */
   enableOpenDebuggerRedirect: boolean,
+
+  /**
+   * Enables the new JS debugger network panel and network CDP events.
+   * When disabled, all CDP events and the network panel are disabled.
```

Comment:
I think Moti is suggesting that we make this message more generic. In particular, "network CDP events" won't be affected by this flag (at least here in the default RN implementation of `InspectorProxy`).

```suggestion
   * Enables the Network panel when launching the custom debugger frontend.
```